common/lanelet2_extension/docs/RegulatoryElements.md

```diff
@@ -2,6 +2,53 @@
 
 To better support dynamic updates and complex situations, CARMA expands on the standard Lanelet2 regulatory element set. The new regulatory element class definitions can be found [here](../include/lanelet2_extension/regulatory_elements) along with a new TrafficRules object which interprets them in place of the Lanelet2 tagging specification.
 
+## Digital Minimum Gap
+
+Minimum Gap can be set dynamically either through a V2V or V2X communications service. It helps to keep a safety following distance between ADS vehicles within an active geofence.
+
+### Parameters
+| **Role** | **Possible Type** | **description**                |
+|-------------|--------------|----------------------------------|
+| **refers**    | **Lanelet, Area**    | The region of the roadway this minimum gap applies to |
+
+### Custom Attributes
+
+| **Key** | **Value Type** | **description**                |
+|-------------|--------------|----------------------------------|
+| **subtype** | **digital_minimun_gap**    | Subtype name |
+| **mingap** | **double**    | The minimum gap to set. In an osm file units must be one of the following m/s, mps, km/h, kmh, m/h, mph |
```

Comment:
Units should be meters right?

Reply:
https://github.com/usdot-fhwa-stol/autoware.ai/pull/145/commits/8f6c38d08ae86e738a2bc7aaa9a91ccc1a60b02c